claim-check/claim-check-producer.bal

```diff
@@ -0,0 +1,48 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/uuid;
+import ballerinax/kafka;
+import ballerina/mime;
+
+final http:Client awsS3Client = check new("http://bucket.s3.amazonaws.com.balmock,io");
+final kafka:Producer kafkaProducer = check new(kafka:DEFAULT_URL);
+
+type ScanRequest [string, stream<byte[], io:Error?>];
```

Comment:
Let's use a record since accessing by index OR use inline type and immediately destructure.